common/src/test/java/org/keycloak/common/util/NetworkUtilsTest.java

```diff
@@ -0,0 +1,27 @@
+package org.keycloak.common.util;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.Timeout;
+import org.keycloak.common.util.NetworkUtils;
+
+import java.lang.reflect.Method;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+public class NetworkUtilsTest {
+
+  @Rule
+  public ExpectedException thrown = ExpectedException.none();
+  @Rule
+  public Timeout globalTimeout = new Timeout(10000);
```

Comment:
Both these rules and their associated imports can be removed from this test class